addons/payment_buckaroo/views/payment_buckaroo_templates.xml

```diff
@@ -1,27 +1,24 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
-    <data noupdate="1">
-        <template id="buckaroo_form">
-            <div>
-                <input type="hidden" name="data_set" t-att-data-action-url="tx_url" data-remove-me=""/>
-                <input type="hidden" name="Brq_websitekey" t-att-value="Brq_websitekey"/>
-                <input type="hidden" name="Brq_amount" t-att-value="Brq_amount"/>
-                <input type="hidden" name="Brq_currency" t-att-value="Brq_currency"/>
-                <input type="hidden" name="Brq_invoicenumber" t-att-value="Brq_invoicenumber"/>
-                <input type="hidden" name="Brq_signature" t-att-value="Brq_signature"/>
-                <input type="hidden" name="brq_test" t-att-value="brq_test"/>
-                <input type="hidden" name="Brq_culture" t-att-value="Brq_culture"/>
-                <!-- URLs -->
-                <input t-if="Brq_return" type="hidden" name='Brq_return'
-                    t-att-value="Brq_return"/>
-                <input t-if="Brq_returncancel" type="hidden" name='Brq_returncancel'
-                    t-att-value="Brq_returncancel"/>
-                <input t-if="Brq_returnerror" type="hidden" name='Brq_returnerror'
-                    t-att-value="Brq_returnerror"/>
-                <input t-if="Brq_returnreject" type="hidden" name='Brq_returnreject'
-                    t-att-value="Brq_returnreject"/>
-                <input type="hidden" name='add_returndata' t-att-value="add_returndata"/>
-            </div>
-        </template>
-    </data>
+
+    <template id="redirect_form">
+        <form t-att-action="tx_url" method="post">
+            <input type="hidden" name="Brq_websitekey" t-att-value="Brq_websitekey"/>
+            <input type="hidden" name="Brq_amount" t-att-value="Brq_amount"/>
+            <input type="hidden" name="Brq_currency" t-att-value="Brq_currency"/>
+            <input type="hidden" name="Brq_invoicenumber" t-att-value="Brq_invoicenumber"/>
+            <input type="hidden" name="Brq_signature" t-att-value="Brq_signature"/>
+            <input t-if="Brq_culture" type="hidden" name="Brq_culture" t-att-value="Brq_culture"/>
+            <!-- URLs -->
+            <input t-if="Brq_return" type="hidden" name='Brq_return'
+                t-att-value="Brq_return"/>
+            <input t-if="Brq_returncancel" type="hidden" name='Brq_returncancel'
+                t-att-value="Brq_returncancel"/>
+            <input t-if="Brq_returnerror" type="hidden" name='Brq_returnerror'
+                t-att-value="Brq_returnerror"/>
+            <input t-if="Brq_returnreject" type="hidden" name='Brq_returnreject'
+                t-att-value="Brq_returnreject"/>
```

Comment:
Alphabetical order of inputs (or following the API documentation order ?)

Reply:
doc order